tsl/test/expected/dist_hypertable-11.out

```diff
@@ -552,6 +552,61 @@ FROM disttable;
  2.7
 (1 row)
 
+EXPLAIN (VERBOSE, COSTS FALSE)
+SELECT max(time)
+FROM disttable;
+                                                                                                       QUERY PLAN                                                                                                       
+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+ Result
+   Output: $0
+   InitPlan 1 (returns $0)
+     ->  Limit
+           Output: disttable."time"
+           ->  Custom Scan (AsyncAppend)
+                 Output: disttable."time"
+                 ->  Merge Append
+                       Sort Key: disttable_1."time" DESC
+                       ->  Custom Scan (DataNodeScan) on public.disttable disttable_1
+                             Output: disttable_1."time"
+                             Data node: db_dist_hypertable_1
+                             Chunks: _dist_hyper_1_4_chunk, _dist_hyper_1_1_chunk
+                             Remote SQL: SELECT "time" FROM public.disttable WHERE _timescaledb_internal.chunks_in(public.disttable.*, ARRAY[2, 1]) AND (("time" IS NOT NULL)) ORDER BY "time" DESC NULLS FIRST LIMIT 1
+                       ->  Custom Scan (DataNodeScan) on public.disttable disttable_2
+                             Output: disttable_2."time"
+                             Data node: db_dist_hypertable_2
+                             Chunks: _dist_hyper_1_5_chunk, _dist_hyper_1_3_chunk
+                             Remote SQL: SELECT "time" FROM public.disttable WHERE _timescaledb_internal.chunks_in(public.disttable.*, ARRAY[2, 1]) AND (("time" IS NOT NULL)) ORDER BY "time" DESC NULLS FIRST LIMIT 1
+                       ->  Custom Scan (DataNodeScan) on public.disttable disttable_3
+                             Output: disttable_3."time"
+                             Data node: db_dist_hypertable_3
+                             Chunks: _dist_hyper_1_6_chunk, _dist_hyper_1_2_chunk
+                             Remote SQL: SELECT "time" FROM public.disttable WHERE _timescaledb_internal.chunks_in(public.disttable.*, ARRAY[2, 1]) AND (("time" IS NOT NULL)) ORDER BY "time" DESC NULLS FIRST LIMIT 1
+(24 rows)
+
+-- Test that Remote Query (obtained in above output) on DN directly uses ChunkAppend plan
+\c :DATA_NODE_3
```

Comment:
There's a GUC, `timescaledb.enable_remote_explain` you can set to show the remote query plan for each data node as part of the access node explain. It might be better to use that in the original AN explain so that we know for sure that the right DN plan is executed when we do the query from the AN. If it gets messy, you can include an extra condition to hit, e.g., just one data node.

Another option is to include `\ir include/remote_exec.sql` which gives you the function `test.remote_exec()` that you can use in the text to execute statements on DNs from the AN. This avoids having to connect to the DN in a new session. Creating new sessions in tests make it harder to run the script in, e.g., a debugger if one should need to do that.

Reply:
Thanks. Used the `timescaledb.enable_remote_explain` GUC and removed all connections to datanode etc.